disco_aws_automation/disco_dynamodb.py

```diff
@@ -132,3 +141,153 @@ def _convert_table_to_dict(table):
                 "StreamSpecification": table.stream_specification,
                 "LatestStreamLabel": table.latest_stream_label,
                 "LatestStreamArn": table.latest_stream_arn}
+
+
+class AsiaqDynamoDbBackupManager(object):
+    """
+    Management tool for creating, restoring from, and inspecting DynamoDB table backups.
+    """
+    S3_BUCKET_KEY = "dynamodb-backup"
+    BACKUP_DESC_TEMPLATE = "Periodic backup for %s table in %s env"
+
+    BACKUP_PIPELINE_TEMPLATE = 'dynamodb_backup'
+    RESTORE_PIPELINE_TEMPLATE = 'dynamodb_restore'
```

Comment:
Feels like we should take these template names as arguments from the user running the command, especially considering we are using them as tags to version control the pipeline.